hudi-common/src/main/java/org/apache/hudi/common/model/BaseAvroPayload.java

```diff
@@ -34,7 +35,7 @@ public abstract class BaseAvroPayload implements Serializable {
   /**
    * Avro data extracted from the source converted to bytes.
    */
-  protected final byte[] recordBytes;
+  protected final ByteArrayOutputStream recordBytesStream;
```

Comment:
Is `ByteArrayOutputStream` serializable?

Reply:
updated, ptal 🌹